api/server.js

```diff
@@ -24,6 +25,9 @@ serverChecks.check(db.serverUrl).then(() => {
   .then(() => console.log('Loading configuration…'))
   .then(config.load)
   .then(() => console.log('Configuration loaded successfully'))
+  .then(() => console.log('Uploading forms'))
+  .then(config.uploadStandardForms)
```

Comment:
This is going to do a bunch of work on api start every time when most of the time the forms will already exist. Can we instead watch for ddoc changes and only extract and upload then? Alternatively we could store the hash of the forms.zip (available from couchdb) so we know if it's changed from the last time we ran it.